x-pack/plugins/triggers_actions_ui/public/application/context/alerts_context.tsx

```diff
@@ -23,6 +23,7 @@ export interface AlertsContextValue {
   >;
   charts?: ChartsPluginSetup;
   dataFieldsFormats?: DataPublicPluginSetup['fieldFormats'];
+  metadata?: Record<string, any>;
```

Comment:
Nit: I would love to be able to pass in a generic here: `AlertContextValue<MetaDataType>` to get some easier type safety.